content/scripts/build.js

```diff
@@ -1825,7 +1843,13 @@ class Builder {
    */
   processFolderTitle(source, folder, allPopularities) {
     const { metadata } = getContent(source, folder, true);
-    const mdn_url = buildMDNUrl(metadata.locale, metadata.slug);
+    let mdn_url;
+    try {
+      mdn_url = buildMDNUrl(metadata.locale, metadata.slug);
+    } catch (err) {
+      console.warn(`The folder that caused the error was: ${folder}`);
+      throw err;
+    }
```

Comment:
Explanation for this change:
First of all, it's off topic and not really related to the issue. Shame on me!
However, I find this really useful. 
When I was fumbling with my testing and messing with `index.html` files, I once accidentally messed up and created a `index.html` with front-matter that didn't have a `slug: ...` and you get a weird error. The error is still "good" in that it's necessary. What's so annoying about it is that you have no idea which folder it was that caused this. So, all this change does is that it "pauses the exception", logs the folder what was the context, and proceeds to throw again.